include/deal.II/lac/trilinos_block_sparse_matrix.h

```diff
@@ -563,7 +563,73 @@ namespace TrilinosWrappers
       BaseClass::vmult_nonblock_nonblock (dst, src);
   }
 
-}
+  namespace internal
+  {
+    namespace BlockLinearOperator
+    {
+//      template<typename PayloadBlockType>
+//      using TrilinosBlockPayload = ::internal::BlockLinearOperator::EmptyBlockPayload<PayloadBlockType>;
+
+      template<typename PayloadBlockType>
+      class TrilinosBlockPayload
```

Comment:
same as for the empty payload class: a docstring would be useful; please use doxygen-style comments for the members